test/e2e/snaps/test-snap-txinsights-v2.spec.js

```diff
@@ -0,0 +1,138 @@
+        await driver.delay(1000);
+
+        // switch to metamask extension and click connect
+        let windowHandles = await driver.waitUntilXWindowHandles(
+          2,
+          1000,
+          10000,
+        );
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({
+          text: 'Connect',
+          tag: 'button',
+        });
+
+        await driver.waitForSelector({ text: 'Install' });
+
+        await driver.clickElement({
+          text: 'Install',
+          tag: 'button',
+        });
+
+        await driver.waitForSelector({ text: 'OK' });
+
+        await driver.clickElement({
+          text: 'OK',
+          tag: 'button',
+        });
+
+        // switch to test-snaps page and get accounts
+        await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
+        await driver.clickElement('#getAccounts');
+        await driver.delay(1000);
+
+        // switch back to MetaMask window and deal with dialogs
+        windowHandles = await driver.waitUntilXWindowHandles(2, 1000, 10000);
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({
+          text: 'Next',
+          tag: 'button',
+        });
+        await driver.delay(1000);
+        await driver.clickElement({
+          text: 'Connect',
+          tag: 'button',
+        });
+
+        // switch to test-snaps page and send tx
+        windowHandles = await driver.waitUntilXWindowHandles(1, 1000, 10000);
+        await driver.switchToWindowWithTitle('Test Snaps', windowHandles);
+        await driver.clickElement('#sendInsights');
+        await driver.delay(1000);
+
+        // switch back to MetaMask window and switch to tx insights pane
+        windowHandles = await driver.waitUntilXWindowHandles(2, 1000, 10000);
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.delay(1000);
+        await driver.clickElement({
+          text: 'Insights Example Snap',
+          tag: 'button',
+        });
+
+        // check that txinsightstest tab contains the right info
+        await driver.waitForSelector({
+          css: '.snap-ui-renderer__content',
+          text: 'ERC-20',
+        });
+
+        // click confirm to continue
+        await driver.clickElement({
+          text: 'Confirm',
+          tag: 'button',
+        });
+
+        // check for warning from txinsights
+        await driver.waitForSelector({
+          css: '.snap-delineator__header__text',
+          text: 'Warning from Insights Example Snap',
+        });
+
+        // click the warning confirm checkbox
+        await driver.clickElement('.mm-checkbox__input');
+
+        // click confirm button to send transaction
+        await driver.clickElement({
```

Comment:
Can we verify that the tx is added to the activity screen?

Reply:
added in latest commit